.github/workflows/release-oci-mcpo.yml

```diff
@@ -0,0 +1,151 @@
+# Stage OCI container images through GitHub Actions (GHA) to GitHub Container Registry (GHCR).
+# The image is available per `ghcr.io/crate/cratedb-mcp`.
+name: OCI MCPO
+
+on:
+
+  # Build images when merging to `main` and when running a release.
+  push:
+    branches:
+      - 'main'
+    tags:
+      - '*.*.*'
+
+  # Build images on each pull request.
+  # Remark: Activate only when needed, it will eat a lot of resources. The alternative
+  #         is to build manually / on-demand, by using the `workflow_dispatch` feature,
+  #         available through the GHA web UI.
+  pull_request:
+
+  # Produce a nightly image every day at 6 a.m. CEST.
+  schedule:
+    - cron: '0 4 * * *'
```

Comment:
Do we need this if we produce an image when merging to `main`?

Reply:
It is not required, but we can optionally have it to verify builds don't go south. Currently, the OCI building operations are "full-on", also building on each and every PR. This can be tuned down any time as we go.